readme.md

```diff
@@ -106,7 +123,22 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+// app.get('/', function (req, res) {
+  res.send('/')
+  });
+
+  app.post('/', function (req, res) {
+    res.send('/')
+    });
+
+    app.put('/', function (req, res) {
+      res.send('/')
+      });
+
+      app.delete('/', function (req, res) {
+        res.send('/')
+        });
+}
 
 ```
 
```

Comment:
You got the idea. The question, however, is looking for you to send a particular string for each response. So something like this...

```js
app.get("/", function(request, response){
  response.send("SHOW");
});
app.post("/", function(request, response){
  response.send("CREATE");
});
app.put("/:id", function(request, response){
  response.send("UPDATE");
});
app.delete("/:id", function(request, response){
  response.send("DELETE");
});
```